kubernetes/helm/swissgeol-assets/templates/deployment.api.yaml

```diff
@@ -74,12 +74,14 @@ spec:
             value: {{ .Values.oauth_token_endpoint }}
           - name: OAUTH_AUTHORIZED_GROUPS
             value: {{ .Values.oauth_authorized_groups }}
+          - name: ANONYMOUS_MODE
+            value: "{{ .Values.anonymous_mode }}"
         resources:
           limits:
             cpu: '0.6'
             memory: 1Gi
           requests:
             cpu: '0.6'
-            memory: 1Gi
+            memory: 500Mi
```

Comment:
I think we can remove this to allow the container to not take up resources while its not actively syncing.

Reply:
this is not the sync container but assets application
We reduced this so we could deploy at all - still questionable whether this is still necessary